src/main/java/com/microsoft/aad/msal4j/MultiTenantAccount.java

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.aad.msal4j;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
+public class MultiTenantAccount extends Account implements IMultiTenantAccount {
+
+    private Map<String, ITenantProfile> tenantProfiles;
```

Comment:
as far As I see tenantProfiles are never set - so always null

Reply:
They are set in TokenCache (line 345), though it uses the setter generated by Lombok. I had an explicit getTenantProfiles() just because it was in the spec, but looking at it now I think it'd be better to have either an explicit getter/setter and not use Lombok here, or to just let both be handled by Lombok. Do you have any preference?